crates/proof-of-sql/src/sql/proof_exprs/sign_expr_test.rs

```diff
@@ -33,20 +32,20 @@ fn prover_evaluation_generates_the_bit_distribution_of_a_negative_constant_colum
     assert_eq!(builder.bit_distributions(), [dist]);
 }
 
-#[test]
-fn count_fails_if_a_bit_distribution_is_out_of_range() {
-    let dists = [BitDistribution::new::<Curve25519Scalar, _>(&[
-        Curve25519Scalar::from(3) * Curve25519Scalar::from(u128::MAX),
-    ])];
-    let mut builder = CountBuilder::new(&dists);
-    assert!(count_sign(&mut builder).is_err());
-}
+// #[test]
```

Comment:
Shall this test be removed?

Reply:
Yes. I removed it and added another to replace it.